src/components/victory-area/helper-methods.js

```diff
@@ -57,7 +57,7 @@ const getBaseProps = (props, fallbackProps) => {
   };
   return data.reduce((childProps, datum, index) => {
     const text = LabelHelpers.getText(props, datum, index);
-    if (text !== undefined && text !== null || events || sharedEvents) {
+    if (text !== undefined && text !== null || (labels && events || sharedEvents)) {
```

Comment:
this is getting a _little_ tough to read. many permutations. we could have separate variables, or group with parents, or comments